app/docker/nginx.conf

```diff
@@ -30,9 +30,10 @@ http {
         location / {
             index  index.html index.htm;
             try_files $uri $uri/ /index.html$is_args$args;
+        }
 
-            add_header Cache-Control "private, max-age=63072000" always;
-            add_header Pragma "private, max-age=63072000" always;
+        location /assets/ {
+            add_header Cache-Control "public, max-age=31536000, immutable" always;
         }
```

Comment:
A summary of why we need to apply the 2 year cache to assets would be good for the rest of the team, just so were clear as to why the original change was made.